admin/load.php

```diff
@@ -540,21 +543,29 @@ function perflab_enqueue_modules_page_scripts() {
  */
 function perflab_activate_plugin() {
 	// Do not proceed if plugin query arg is not present.
-	if ( empty( $_GET['plugin'] ) ) {
+	if ( empty( $_GET['plugin_slug'] ) ) {
 		return;
 	}
 
-	// The plugin being activated.
-	$plugin = sanitize_text_field( wp_unslash( $_GET['plugin'] ) );
+	$plugin_slug = sanitize_text_field( wp_unslash( $_GET['plugin_slug'] ) );
 
-	check_admin_referer( "perflab_activate_plugin_{$plugin}" );
+	check_admin_referer( "perflab_activate_plugin_{$plugin_slug}" );
 
-	if ( ! current_user_can( 'activate_plugin', $plugin ) ) {
+	$plugins = get_plugins( '/' . $plugin_slug );
+
+	if ( empty( $plugins ) ) {
+		return;
+	}
+
+	$keys        = array_keys( $plugins );
```

Comment:
I suggest something more descriptive than `$keys`. What does this array actually contain?

Reply:
It's the "plugin file name" per `get_plugins()` so probably better to name this `$plugin_file_names`